### DISCUSSION TITLE
blog/2024/mirroring-in-sr-linux/

### DISCUSSION THREAD

**Discussion by giscus** [srl-labs/learn-srlinux#159] [comments]
# blog/2024/mirroring-in-sr-linux/

Open documentation for open Network OS

https://learn-srlinux.pages.dev/blog/2024/mirroring-in-sr-linux/

**Comment by pstavirs** [comment]
Want to go beyond ICMP ping?

Use an [Ostinato node](https://containerlab.dev/manual/kinds/ostinato/) instead of the two clients and you can  try a variety of different protocol traffic at various rates including but not limited to multicast and broadcast.

![](https://drive.google.com/uc?export=view&id=1CvyWNXa4DLwJCRhQ4DX3W0LWk9YsR1xY)
